portfolio/src/main/java/com/google/sps/data/BlogMessage.java

```diff
@@ -59,8 +63,10 @@ public String getSender() {
     return messageReplies;
   }
 
-  public void addReply(BlogMessage message) {
-    messageReplies.add(message);
+  public void addReply(BlogMessage reply) {
```

Comment:
Since this just adds a reply to messageReplies, i think it's not needed?
You could use the getReplies function to get the list and add to it. 
Like post.getReplies().add(reply).

Reply:
This would be correct if addReply were a utility function. However, it is usually good Object Oriented design to keep member variables private whenever possible. In this case using the 'setter' addReply is a good choice because it allows users of BlogMessage to add replies to a post, but it prevents them from doing whatever they want with messageReplies.